src/index.js

```diff
@@ -1,13 +1,22 @@
-import './config.js';
 import express from 'express';
 
-import { router } from './api/index.js';
+import { usersRouter, groupsRouter } from './api/index.js';
 
 const PORT = Number(process.env.PORT) || 3000;
 const app = express();
 
 app.use(express.json());
-app.use('/', router);
+
+app.use('/users', usersRouter);
+app.use('/groups', groupsRouter);
+
+app.use((error, req, res, next) => {
```

Comment:
did you check that middleware? I have an assumption that is does not catch exceptions

Reply:
I've checked it, and it does seem to catch exceptions. Why do you assume it wouldn't?

Reply:
yes, my bad, it works